Dockerfile

```diff
@@ -14,5 +14,5 @@ COPY . /app/atd-kits
 RUN chmod -R 755 /app/*
 
 ## Proceed to install the requirements...do
-RUN cd /app/atd-kits && apt-get update && \
+RUN cd /app/atd-kits && \
```

Comment:
This isn't part of the solution, but we don't need to update the debian repository manifests to do the `pip install`.